Source/Initialization/WarpXInitData.cpp

```diff
@@ -156,6 +319,8 @@ WarpX::InitData ()
     // (example: a box with 16 valid cells and 32 guard cells in z will not be considered valid)
     CheckGuardCells();
 
+    PrintMainPICparameters();
```

Comment:
@peterscherpelz @roelof-groenewald @MKieburtz looking at #2033, would you prefer to print this only when `verbose` is set? (default: yes, but off in your Python runs)

Reply:
@ax3l Thanks for checking, I would vote for outputting all of this since it's only printed once per simulation. The main thing we wanted to cut out with verbose was per-step prints, since we mostly have runs going to millions of steps.